apps/02RockPaperScissors.js

```diff
@@ -6,11 +6,49 @@ prompt.start();
 
 
 function rockPaperScissors(hand1, hand2) {
+if (hand1 === hand2)  {
+  return "It\'s a tie!"
+}
 
-    // Write code here
+else if (
+  (hand1 === 'rock' && hand2 === 'scissors') ||
+  (hand1 === 'scissors' && hand2 === 'paper') ||
+  (hand1 === 'paper' && hand2 === 'rock')
+) {
+  return "Hand one wins!";
+}
 
+else {
+  return "Hand two wins!";
 }
 
```

Comment:
Beautiful! Love the use of "or." 

![over9](https://cdn2.scratch.mit.edu/get_image/gallery/1619018_200x130.png?v=1444926215.39)